securedrop_client/locale/messages.pot

```diff
@@ -35,7 +35,7 @@ msgstr ""
 msgid "That didn't work. Please check everything and try again."
 msgstr ""
 
-msgid "You must sign in to perform this action."
+msgid "You must sign-in to perform this action."
```

Comment:
Not terribly confident about this. The goal is to make my tool not complain about `in to` (where it would suggest `into`).

Reply:
Mmm, it seems like you've got a new valid sentence to handle in `check-spelling` :wink: I don't think that hyphen belongs there.

Reply:
Oh, that isn't check-spelling, that's me, I'm a human who still struggles with things. `check-spelling` can _optionally_ suggest that ` in to ` is sometimes wrong, but you can choose not to include that pattern or choose to give it more hints about things to ignore. The problem is that it operates on patterns, it can't "lex" English (or some other language for that matter), so it can't know that "perform" is a verb and thus that the construction is valid. -- I probably should just switch to asking grammarly.com for input when I run into sentences with `in to`....

Anyway, I'll drop it.